src/Sylius/Bundle/ApiBundle/Resources/config/services/command_handlers.xml

```diff
@@ -237,5 +237,21 @@
             <tag name="messenger.message_handler" bus="sylius.command_bus" />
             <tag name="messenger.message_handler" bus="sylius_default.bus" />
         </service>
+
+        <service id="Sylius\Bundle\ApiBundle\CommandHandler\Payment\AddPaymentRequestHandler">
+            <argument type="service" id="sylius.custom_factory.payment_request" />
+            <argument type="service" id="sylius.repository.payment_method" />
+            <argument type="service" id="sylius.repository.payment" />
+            <argument type="service" id="Sylius\Bundle\CoreBundle\PaymentRequest\CommandDispatcher\PaymentRequestCommandDispatcherInterface" />
+            <tag name="messenger.message_handler" bus="sylius.command_bus" />
+            <tag name="messenger.message_handler" bus="sylius_default.bus" />
```

Comment:
Wondering if we should add a custom transport for payment processing only. A bit more to configure and remember, but having a distinct and independent consumer might be a nice thing to have.

Reply:
yes it can ease the retry of failed messages too.
I will figure a name for this.